### PR TITLE
add unit test for conn_pool.go and tx.go 

### PR DESCRIPTION
**Pull request by jasondeng1997**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/conn_pool_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package runtime
```
Comment (Mulavar) [inline pkg/runtime/conn_pool_test.go:1]: please add apache license.

File: pkg/runtime/tx_test.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	"github.com/arana-db/arana/pkg/mysql"
+	"github.com/arana-db/arana/pkg/proto"
+	"github.com/arana-db/arana/pkg/runtime/gtid"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/atomic"
+	"testing"
+	"time"
+)
+
+func Test_branchTx_CallFieldList(t *testing.T) {
```
Comment (jjeffcaii) [inline pkg/runtime/tx_test.go:33]: The test method name looks weird.

File: pkg/runtime/tx_test.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+			got, err := tx.CallFieldList(tt.args.ctx, tt.args.table, tt.args.wildcard)
+			if !tt.wantErr(t, err, fmt.Sprintf("CallFieldList(%v, %v, %v)", tt.args.ctx, tt.args.table, tt.args.wildcard)) {
+				return
+			}
+			assert.Equalf(t, tt.want, got, "CallFieldList(%v, %v, %v)", tt.args.ctx, tt.args.table, tt.args.wildcard)
+		})
+	}
+}
+
+func Test_compositeTx_Rollback(t *testing.T) {
```
Comment (jjeffcaii) [inline pkg/runtime/tx_test.go:77]: The test method name looks weird.

File: pkg/runtime/tx_test.go [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    proto.Result
+		want1   uint16
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
```
Comment (jjeffcaii) [inline pkg/runtime/tx_test.go:102]: Finish the unit test body, or it looks meaningless.

File: pkg/runtime/tx_test.go [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+		wildcard string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    []proto.Field
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
```
Comment (jjeffcaii) [inline pkg/runtime/tx_test.go:55]: Finish the unit test body, or it looks meaningless.

**Comment by jjeffcaii** [comment]
Since it's been a long time, and it only affects the unit tests, I've merged it. Thanks!

**Pull request merged**